wheel/python/clvm_rs/at.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+from typing import Optional, cast
+
+from .clvm_storage import CLVMStorage
+
+
+def at(obj: CLVMStorage, position: str) -> Optional[CLVMStorage]:
+    """
+    Take a string of `f` and `r` characters and follow that path.
+
+    Example:
+
+    ```
+    p1 = Program.to([10, 20, 30, [15, 17], 40, 50])
+    assert Program.to(17) == at(p1, "rrrfrf")
+    ```
+
+    Returns `None` if an atom is hit at some intermediate node.
+
+    ```
+    p1 = Program.to(10)
+    assert None == at(p1, "rr")
+    ```
+
+    """
+    v: Optional[CLVMStorage] = obj
```

Comment:
why is `v` an `Optional`? it seems like it's always set.

Reply:
You're right, removing.